exts/cesium.usd.plugins/schemas/cesium_schemas.usda

```diff
@@ -413,12 +413,45 @@ class CesiumTilesetPrim "CesiumTilesetPrim" (
     )
 }
 
-class CesiumImageryPrim "CesiumImageryPrim" (
-    doc = """Adds a prim for representing an imagery layer. Should be a child of a tileset."""
+class "CesiumImageryPrim" (
+    doc = """Abstract base class for prims that represent an imagery layer."""
     inherits = </Typed>
     customData = {
         string className = "Imagery"
     }
+) {
+    bool cesium:showCreditsOnScreen = false (
+        customData = {
+            string apiName = "showCreditsOnScreen"
+        }
+        displayName = "Show Credits On Screen"
+        doc = "Whether or not to show this imagery layer's credits on screen."
+    )
+
+   float cesium:alpha = 1.0 (
+        customData = {
+            string apiName = "alpha"
+        }
+        displayName = "Alpha"
+        doc = "The alpha blending value, from 0.0 to 1.0, where 1.0 is fully opaque."
+    )
+
+    uniform token cesium:overlayRenderPipe = "overlay" (
```

Comment:
The name "overlayRenderPipe" doesn't sound quite right to me, maybe "overlayRenderType" or "overlayRenderMethod"?

Reply:
I was debating between "Pipe" and "Method". Let's go with `overlayRenderMethod`. That way there isn't an `overlayType` and `overlayRenderType`.